ui/app/src/components/Navigation/PathNavigator/PathNavigator.tsx

```diff
@@ -369,15 +427,13 @@ export default function PathNavigator(props: WidgetProps) {
         onCloseItemMenu={onCloseItemMenu}
         onCloseSimpleMenu={onCloseSimpleMenu}
         onSimpleMenuClick={onSimpleMenuClick}
-        onItemMenuActionSuccessCreator={onItemMenuActionSuccessCreator}
       />
     </Suspencified>
   );
 }
 
 export function PathNavigatorUI(props: WidgetUIProps) {
   const classes = useStyles();
-  // region consts {...} = props
```

Comment:
Leave region unless it ends up being a single line. If the region is removed, the endregion should be too.